server/routes/api/event_logger.js

```diff
@@ -0,0 +1,25 @@
+const express = require('express')
+const router = express.Router()
+const logger = require('../../utilities/winston_logger')
+const ErrorLog = require('../../db/models/error_log')
+
+router.post('/log', async (req, res) => {
+  const { severity, data } = req.body
+  // send log messages to Winston
+  logger.info(JSON.stringify({ severity, data }))
+  res.sendStatus(200)
+
+  // save log messages to database
+  try {
+    await ErrorLog.query().insert({
```

Comment:
I am not sure that this is the correct way to insert because when I run your tests for this, I always get a 500 response. Look at `checkout.js` for an example where you import `createClient` and then can do something like `createClient('error_logs').insert(...)`.

Remember to add a successful response code. Right now, that request will either return 500 or hang until it times out.

One other small critique: although it is not necessarily consistent through this project, it is best practice to return a reponse with `return res(xxx).anyOtherChainedMethods.end()`. That will make sure the function exits and closes the response. Otherwise, you could continue to run code in that endpoint after you meant to exit, which can cause a bunch of problems.